src/server.ts

```diff
@@ -15,65 +15,42 @@
  */
 
 import * as express from 'express';
-import * as fs from 'fs';
+import * as path from 'path';
 import * as morgan from 'morgan';
 import * as url from 'url';
-
+import exhbs = require('express-handlebars');
 import * as datastore from './lib/datastore';
-
 import cookieSession = require('cookie-session');
 import * as github from './lib/github';
 import {config} from './lib/config';
 import {totpCode} from './lib/totp-code';
 import * as request from 'request';
 import {require2fa} from './lib/packument';
 import uuid = require('uuid');
-import * as path from 'path';
 import {json} from './lib/json';
 import {drainRequest} from './lib/drain-request';
-
 const validatePackage = require('validate-npm-package-name');
-
 import {publish} from './routes/publish';
 import {putDeleteTag} from './routes/put-delete-tag';
+const unsafe = require('./lib/unsafe.js');
 
 const ONE_DAY = 1000 * 60 * 60 * 24;
-const unsafe = require('./lib/unsafe.js');
-const app = express();
 
-const readStatic = (p: string) => {
-  return fs.readFileSync(path.resolve(__dirname + '/../../html', p));
-};
+const app = express();
 
-const ghcss = fs.readFileSync(
-  require.resolve('github-markdown-css/github-markdown.css')
+app.engine(
+  '.hbs',
+  exhbs({
+    extname: '.hbs',
+    defaultLayout: 'main',
+    partialsDir: path.join(__dirname, '../../views/partials'),
+    layoutsDir: path.join(__dirname, '../../views/layouts'),
+  })
 );
-const favicon = readStatic('favicon.ico');
-const changelog = readStatic('changelog.html')
-  .toString('utf8')
-  // Reduce the heading level in the CHANGELOG by 1:
-  .replace(/h3/g, 'h4')
```

Comment:
Just a heads up - this down-headerizing is missing from this solution.  Not sure how much we really cared about it.

Reply:
The CHANGELOG being embedded in the page starts to look a little overwhelming with these headers, and the fact that we have multiple `h1` tags breaks HTML best practices.

I think embedding the CHANGELOG is neat, but it does start to dominate that page, I wonder if there's a better approach, perhaps we should just link to it?

Reply:
I'm happy to re-do the header downsizing, but agree that putting it on it's own page likely makes more sense.  You cool with linking out to it?

Reply:
Yes let's just link to it.